config/vufind/RateLimiter.yaml

```diff
@@ -157,5 +157,8 @@ Policies:
   # siteKey: 0x1234567890
   # secretKey: 0x1234567890
 
-  # Verify API URL.  Default is https://challenges.cloudflare.com/turnstile/v0/siteverify
+  # Verify API URL. Default is below.
   # verifyUrl: https://challenges.cloudflare.com/turnstile/v0/siteverify
+
+  # API Javascript file. Default is below.
```

Comment:
Since we're relying on a third-party Javascript file, should we include a note about adjusting the content security policy to allow this (and/or include a commented-out example in contentsecuritypolicy.ini, if relevant)?

Reply:
Thanks for the reminder, I had done this locally but forgot to add to the PR.  There is no CSP rule needed for this javascript file or for the inline script, since both already have the nonce attached for CSP v3, and for CSP v2 the config file is already allowing all scripts (which is honestly problematic but then so is CSP v2 in general).  I did though need to add a frame-src rule, since strict-dynamic unfortunately doesn't cover that.  Added a comment as well.